design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,212 @@
+type Mutation {
+    createNegotiableQuote(
+        cartId: ID! @doc(description: "Cart ID")
+        quoteName: String @doc(description: "Quote name")
```

Comment:
Believe `quoteName` is a required field, right? If not, this UI would look strange:

![image](https://user-images.githubusercontent.com/5233399/85440184-69ca4c80-b553-11ea-8a64-fa37a5101286.png)